CONTRIBUTING.md

```diff
@@ -1,98 +1,278 @@
-Contributing code
-=================
-
 Things to know when contributing code
 =====================================
-
-  * You agree to license your contributions under [MPL 2][MPL2].
-  * Discuss large changes in [Discourse][discourse]
-    or on a [bugzilla bug][mdn-backlog] before coding.
-  * Python code style should follow [PEP8 standards][pep8] whenever possible.
-  * We don't accept pull requests for translated strings (i.e. anything under locale/).
-    Please use [Pontoon][pontoon] instead.
+* You agree to license your contributions under [MPL 2][MPL2].
+* You agreed to follow the [Code of Conduct][coc].
+* Discuss large changes in [Discourse][discourse] or on a
+  [bugzilla bug][mdn-backlog] before coding.
+* We don't accept pull requests for translated strings (i.e. Anything under
+  locale/). Please use [Pontoon][pontoon] instead.
 
 [MPL2]: http://www.mozilla.org/MPL/2.0/
+[coc]: https://github.com/mozilla/kuma/blob/master/CODE_OF_CONDUCT.md
 [discourse]: https://discourse.mozilla.org/c/mdn
 [mdn-backlog]: https://mzl.la/2onLvZ8
-[pep8]: http://www.python.org/dev/peps/pep-0008/
 [pontoon]: https://pontoon.mozilla.org/projects/mdn/
 
+What to work on
+===============
+The MDN platform is a mature product, which means there aren't always bugs that
+are suitable for new contributors. We keep a list of starting mentored bugs on
+the [MDN "Getting Involved" page](get-involved). There are interesting MDN
+projects outside of writing Python for the platform, but they may be harder
+to find. If you have questions about what to work on, you can ask:
+
+* In the #mdndev [IRC channel on irc.mozilla.org][irc-howto].
+* In the [MDN Topic][discourse] in Discourse.
+
+[get-involved]: https://wiki.mozilla.org/Webdev/GetInvolved/developer.mozilla.org#Mentored_Bugs
+[irc-howto]: https://wiki.mozilla.org/Irc
+
+How to submit code
+==================
+The MDN development process is similar to [GitHub Flow][gh-flow]. The general
+steps are:
+
+1. [Install our development environment][install].
+2. Create a branch for your bug.
+3. Make your changes, and create one or more commits.
+4. Push your branch to your fork.
+5. Open a pull request (PR).
+6. Fix any issues identified by [TravisCI][travisci], our automated testing
+   provider.
+7. Fix any issues identified by code review.
+
+MDN staff will take a look at your PR in 1 to 2 business days, either to review
+and give feedback, or to tell you when we plan to review it.
+
+[gh-flow]: https://guides.github.com/introduction/flow/
+[install]: https://kuma.readthedocs.io/en/latest/installation.html.
```

Comment:
Typo: remove trailing period (should be `https://kuma.readthedocs.io/en/latest/installation.html`)